test/hummingbot/connector/gateway/amm/debug_gateway_evm_amm.py

```diff
@@ -42,7 +42,7 @@ class GatewayEVMAMMDataCollector:
     def __init__(self):
         self._clock: Clock = Clock(ClockMode.REALTIME)
         self._connector: GatewayEVMAMM = GatewayEVMAMM(
-            "uniswap",
+            "amm",
```

Comment:
We have connector / exchange named uniswap
https://hummingbot.org/exchanges/uniswap/

this changes here not needed